packages/devtools-reps/src/reps/function.js

```diff
@@ -74,11 +74,22 @@ function getTitle(grip, props) {
 }
 
 function getFunctionName(grip, props) {
-  let name = grip.userDisplayName
-    || grip.displayName
-    || grip.name
-    || props.functionName
-    || "";
+  let { functionName } = props;
+  let end = functionName.length - 1;
+
+  props.functionName =
+    functionName.indexOf('"') == 0 && functionName.lastIndexOf('"') == end
+      ? functionName.substring(1, end)
+      : functionName;
+
+  let name =
+    grip.displayName != undefined && grip.displayName != props.functionName
```

Comment:
the ternary here looks a bit complex, maybe we could have : 
```js
let name;
if (grip.displayName != undefined && functionName != undefined &&  grip.displayName != functionName) {
  name = functionName + ": " + grip.displayName;
} else {
  name = grip.userDisplayName
    || grip.displayName
    || grip.name
    || functionName
    || "";
}
```

Note that I replace the `props.functionName` by `functionName`, since we do have the variable at hand.
I also added `functionName != undefined` in the first test, so we don't end up with things like `undefined: myDisplayName`.

I'm a bit worried about the `name = functionName + ": " + grip.displayName;`, can you give examples of what it should be like ?

Reply:
I made the changes :) Thanks! :)  jQuery has two properties $ and jQuery each with the same displayName. So, instead of -> 
<img width="1137" alt="issue" src="https://user-images.githubusercontent.com/16086407/32224241-4a57dd62-be49-11e7-88a5-1c613e8f1cc0.png">

Jason Laster proposed to make it show $: jQuery() and jQuery(). So, now after the patch it looks -> 
![screenshot-2017-10-31 debugger](https://user-images.githubusercontent.com/16086407/32224353-c5577504-be49-11e7-921a-4eb4cb9b3b11.png)